### DISCUSSION TITLE
Function like ZbDeviceTopic for Bluetooth sensors

### DISCUSSION THREAD

**Discussion by jeroenvermeulen** [arendst/Tasmota#13867] [Ideas]
I have an ESP32 in my network called `bluetooth-bridge-1` and it currently has 7 bluetooth sensors connected, which will probably increase. Most of them are LYWSD03MMC / ATC running [Custom firmware](https://atc1441.github.io/TelinkFlasher.html).

For my Zigbee bridge I use the setting `ZbDeviceTopic 1` so I get separate MQTT topics for each sensor, for example:
`tele/zbbridge/0296/SENSOR`.

Wouldn't it be a nice feature to make this also possible for Bluetooth sensors?
For example `tele/bluetooth-bridge-1/ATC26d29f/SENSOR`

**Comment by barbudor** [comment]
[MI32Option](https://tasmota.github.io/docs/Commands/#ble-mi-sensors)
`MI32Option6 1` will do what you want except that the topic is not custimizable and will always be `tele/tasmota_ble/XXXXX`

**Comment by jeroenvermeulen** [comment]
Oh wow, that already helps me a lot. The sensor ids are unique anyway, so for me `tele/tasmota_ble/ATC26d29f` is perfectly usable.
Thanks for helping me twice this evening!
